### DISCUSSION TITLE
Possibility to define a binding that adds a tag to messages?

### DISCUSSION THREAD

**Discussion by equaeghe** [wshanks/tbkeys#186] [Q&A]
I am new to TBkeys and am looking at the possibility of using it do define a binding that adds a specific tag to one or more messages. Essentially, I would like to replace the action of going to the tagging menu and there selecting a specific tag with a single keystroke. For  tags, there are some default bindings, 0 to remove all tags and 1-9 to add the 1st-9th tag (as defined by their alphabetic order). I would like to define custom keybindings, for example, repurposing 3 to apply the 12th tag I have defined instead of the 3th one.

**Comment by morat523035** [comment]
Similar issues
http://github.com/wshanks/tbkeys/issues/117
http://github.com/wshanks/tbkeys/issues/198
http://github.com/wshanks/tbkeys/issues/220
